Adding Troubleshooting Info for GH007 Error during gh-gei Migration (#25281)
github/docs: content/migrations/using-github-enterprise-importer/completing-your-migration-with-github-enterprise-importer/troubleshooting-your-migration-with-github-enterprise-importer.md

- [`Subsystem 'sftp' could not be executed` error](#subsystem-sftp-could-not-be-executed-error)
- [`Source export archive... does not exist` error](#source-export-archive-does-not-exist-error)
- [`Repository rule violations found` error](#repository-rule-violations-found-error)

> - [`Git source migration failed` error](#git-source-migration-failed-error)

Reviewer: ```suggestion
- [`Your push would publish a private email address` error](#your-push-would-publish-a-private-email-address-error)
```

You can temporarily disable your rulesets during your migration, or you can use bypass mode or the bypass list to exempt your migration from configured rules. For more information, see "[AUTOTITLE]({% ifversion repo-rules-enterprise %}{% else %}/enterprise-cloud@latest{% endif %}/organizations/managing-organization-settings/managing-rulesets-for-repositories-in-your-organization)."

> ### `Git source migration failed` error

Reviewer: ```suggestion
### `Your push would publish a private email address` error
```

> If you receive a `Git source migration failed` error, such as `GH007: Your push would publish a private email address`, review the email settings of your account. Go to *Settings* > *Emails* and scroll to the checkbox "Block command line pushes that expose my email" and make sure it is unchecked.

Reviewer: ```suggestion
If you receive a `Git source migration failed` error with `GH007: Your push would publish a private email address`,  the Git source you're trying to migrate includes commits authored by an email address that you have blocked from being pushed to {% data variables.product.prodname_dotcom %}. For more information, see "[AUTOTITLE]({% ifversion not fpt or ghec %}/enterprise-cloud@latest{% endif %}/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/blocking-command-line-pushes-that-expose-your-personal-email-address){% ifversion fpt or ghec %}."{% else %}" in the  {% data variables.product.prodname_ghe_cloud %} documentation.{% endif %}

To resolve this error, you can either rewrite the Git history to remove the email address, or you can disable the "Block command line pushes that expose my email" setting. 
```
